acvm-repo/acir_field/src/field_element.rs

```diff
@@ -274,21 +274,33 @@ impl<F: PrimeField> AcirField for FieldElement<F> {
     }
 
     fn to_be_bytes(self) -> Vec<u8> {
-        // to_be_bytes! uses little endian which is why we reverse the output
-        // TODO: Add a little endian equivalent, so the caller can use whichever one
-        // TODO they desire
         let mut bytes = Vec::new();
         self.0.serialize_uncompressed(&mut bytes).unwrap();
         bytes.reverse();
         bytes
     }
 
+    /// Converts the field element to a vector of bytes in little-endian order
+    fn to_le_bytes(self) -> Vec<u8> {
+        let mut bytes = Vec::new();
+        self.0.serialize_uncompressed(&mut bytes).unwrap();
+        bytes
+    }
+
     /// Converts bytes into a FieldElement and applies a
     /// reduction if needed.
     fn from_be_bytes_reduce(bytes: &[u8]) -> FieldElement<F> {
         FieldElement(F::from_be_bytes_mod_order(bytes))
     }
 
+    /// Converts bytes in little-endian order into a FieldElement and applies a
+    /// reduction if needed.
+    fn from_le_bytes_reduce(bytes: &[u8]) -> FieldElement<F> {
+        let mut le_bytes = bytes.to_vec();
+        le_bytes.reverse();
+        FieldElement(F::from_be_bytes_mod_order(&le_bytes))
```

Comment:
I see that there is actually an `F::from_le_bytes_mod_order` method available, and in fact [F::from_be_bytes_mod_order](https://github.com/arkworks-rs/algebra/blob/1971370f8dbbe34d7cf2dde90f71317d959b2da0/ff/src/fields/prime.rs#L63-L66) is implemented the way you do it here, by copying and reversing the bytes. That means your implementation of `from_le_bytes_reduce` ends up reversing it twice. It would be nicer to call `from_le_bytes_mod_order` directly.